test/middleware/is.psc.validation.middleware.unit.ts

```diff
@@ -0,0 +1,51 @@
+jest.mock("../../src/validators/is.psc.validator");
+
+import { Request, Response } from "express";
+import { isPscQueryParameterValidationMiddleware } from "../../src/middleware/is.psc.validation.middleware";
+import { Templates } from "../../src/types/template.paths";
+import { isPscFlagValid } from "../../src/validators/is.psc.validator";
+
+const req: Request = {} as Request;
+const res: Response = {} as Response;
+const mockStatus = jest.fn() as jest.Mock;
+const mockRender = jest.fn() as jest.Mock;
+mockRender.mockImplementation((..._args: any) => { return; });
+mockStatus.mockImplementation((_view: string, _options?: object) => { return { render: mockRender }; });
+res.status = mockStatus;
+const next = jest.fn();
+const mockIsPscValidator = isPscFlagValid as jest.Mock;
+mockIsPscValidator.mockReturnValue(true);
+
+describe("is PSC validation middleware tests", () => {
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it("should call next() when isPsc query parameter not present", () => {
+    req.query = { isPsc: undefined };
```

Comment:
i's also add a test to replicate no query string present (might have to console.log the req.query value in the middleware to see what it looks like when no query string is in url).
so have a test where req.query = {}  ( or req.query = undefined, or both!)

Reply:
One new test added for req.query = {}. req.query = undefined causes an error and doesn't represent what comes through when no query params are present.